src/SDKs/ApiManagement/ApiManagement.Tests/Helpers/Extensions.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Xml.Linq;
+
+namespace ApiManagementManagement.Tests.Helpers
+{
+    public static class Extensions
+    {
+        public static bool DictionaryEqual<TKey, TValue>(this IDictionary<TKey, TValue> first, IDictionary<TKey, TValue> second)
```

Comment:
@solankisamir 
nit: Can you not use Linq extension methods first.Except<String>(second) or better use Linq to do the comparison.
I assume you only want to compare keys and not their values, in that case you can still do that using the above